tests/Unit/CodeAnalysisTest.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Answear\FanCourierBundle\Tests\Unit;
+
+use PHPUnit\Framework\Attributes\Test;
+use PHPUnit\Framework\TestCase;
+
+class CodeAnalysisTest extends TestCase
+{
+    #[Test]
+    public function testNoAnnotationTest(): void
```

Comment:
I'm not sure tests are right place to check the annotation. Can we add this when creating a commit, e.g. using GrumPHP?

Reply:
GrumPHP can be skipped.